frontend/src/pages/view-book.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import axios from 'axios';
+import { URL_VIEW_ALL_BOOK } from '../configs.js';
+import Box from '@mui/material/Box';
+import ImageListItem from '@mui/material/ImageListItem';
+import Typography from '@mui/material/Typography';
+import { Button } from '@mui/material';
+
+const ViewBook = () => {
+  const [book, setBook] = useState([]);
+  const { book_id } = useParams();
+
+  useEffect(() => {
+    const fetchBook = async (id) => {
+      try {
+        const res = await axios.get(`${URL_VIEW_ALL_BOOK}/${id}`);
+        setBook(res.data[0]);
+      } catch (err) {
+        console.log(err);
+      }
+    }
+    fetchBook(book_id);
+  }, [book_id]);
+
+  return (
+    <Box
+      sx={{
```

Comment:
```suggestion
      sx={{
         display: 'flex',
         flexDirection: 'column',
```
so that all the items inside the box can be aligned to the center?